pkgs/applications/science/molecular-dynamics/raspa/default.nix

```diff
@@ -0,0 +1,83 @@
+{ lib
+, stdenv
+, callPackage
+, fetchFromGitHub
+, autoconf
+, automake
+, autoreconfHook
+, makeWrapper
+, fftw
+, lapack
+, openblas
+}:
+stdenv.mkDerivation rec {
+  pname = "raspa";
+  version = "2.0.47";
+  passthru.tests.run-an-example = callPackage
+    (
+      { lib
+      , runCommandLocal
+      , raspa
+      , raspa-data
+      }:
+      runCommandLocal "raspa-test-run-an-example" { }
+        ''
+          set -eu -o pipefail
+          exampleDir="${raspa-data.examples}/share/raspa/examples/Basic/1_MC_Methane_in_Box"
+          exampleDirWritable="$(basename "$exampleDir")"
+          cp -rT "$exampleDir" "./$exampleDirWritable"
+          chmod u+rw -R "$exampleDirWritable"
+          cd "$exampleDirWritable"
+          ${raspa}/bin/simulate
+        ''
+    )
+    { };
```

Comment:
```suggestion
  passthru.tests.run-an-example = runCommandLocal "raspa-test-run-an-example" { }
    ''
      set -eu -o pipefail
      exampleDir="${raspa-data.examples}/share/raspa/examples/Basic/1_MC_Methane_in_Box"
      exampleDirWritable="$(basename "$exampleDir")"
      cp -rT "$exampleDir" "./$exampleDirWritable"
      chmod u+rw -R "$exampleDirWritable"
      cd "$exampleDirWritable"
      ${raspa}/bin/simulate
   '';
```

Reply:
Should we then make `raspa` and `raspa-data` as input parameters of the `raspa` derivation?

Reply:
Done.